view.php

```diff
@@ -33,6 +33,7 @@
 $action     = optional_param('action', '', PARAM_ALPHA);
 $userids    = optional_param_array('userid', array(), PARAM_INT); // array of attempt ids for delete action
 $notify     = optional_param('notify', '', PARAM_ALPHA);
+$maxenrollments     = optional_param('maxenrollments', '', PARAM_INT);
```

Comment:
Do we need this optional_param here? More simply, we could simply use `$choicegroup->maxenrollments` throughout the code.